gtest/src/log.rs

```diff
@@ -18,8 +21,40 @@ impl CoreLog {
             dest: other.dest,
             payload: other.payload.into_raw(),
             exit_code: other.reply.map(|(_, code)| Some(code)).unwrap_or_default(),
+            message_id: other.id
         }
     }
+
+    pub(crate) fn generate_reply(&self, payload: Payload, message_id: MessageId, value: u128) -> Message{
+        Message{
```

Comment:
Run `make fmt` or better `make pre-commit`.

```suggestion
        Message {
```